recipes/common/Makefile.common

```diff
@@ -89,6 +89,7 @@ bootc: quadlet
 	  --build-arg APP_IMAGE=$(APP_IMAGE) \
 	  --build-arg SERVER_IMAGE=$(SERVER_IMAGE) \
 	  --build-arg "SSHPUBKEY=$(SSH_PUBKEY)" \
+	  --build-arg PORT=$(PORT) \
```

Comment:
Is PORT Defined?

I think this would be better as:

  $${PORT:+--build-arg PORT=$${PORT}} \